### PR TITLE
Add focus style for donation form radio buttons

### PR DESCRIPTION
**Pull request by denisw**
Makes focus look the same as hover (orange border). However, if a radio button is focused _and_ active, an orange border would be invisible because of the orange background color; in this case, an orange-red border is used instead.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit ac5f126224d2447f43a7c4920e587fcb01623a3e

https://deploy-preview-138--masifunde.netlify.com

**Comment by torkjels** [comment]
I just tested this out real quick, but when I tab to the project selection (SA or DE), or the interval selection, and then press the arrow keys, I actually select the radio buttons.
But when I do the same on the amount selection, the arrow keys only changes which one has focus, it's not selecting any button.

**Comment by denisw** [comment]
@torkjels Yeah, I forgot to pass through some input props there. Fixed now.

File: components/DonationForm/AmountFormSection.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import { Field } from 'react-final-form'
 import styled from 'styled-components'
 
-import LabelButton from './LabelButton'
+import RadioButton from './RadioButton'
```
Comment (torkjels) [inline components/DonationForm/AmountFormSection.js]: Good name change 👍

File: components/DonationForm/RadioButton.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+`
+
+const Input = styled.input.attrs({ type: 'radio', autoComplete: 'off' })`
+  /* Hide input (copied from boostrap) */
+  position: absolute;
+  clip: rect(0,0,0,0);
+  pointer-events: none;
+`
+
+const RadioButton = ({
```
Comment (torkjels) [inline components/DonationForm/RadioButton.js]: Yeah this is great. Now we don't have to repeat the same `<input>` stuff everywhere we use this! Great!

**Pull request merged**